### DISCUSSION TITLE
I would like to know if the plugin rabbitmq-delayed-message-exchange supports rabbitmq-3.5.6?

### DISCUSSION THREAD

**Discussion by peoplewhoworkhard** [rabbitmq/rabbitmq-delayed-message-exchange#212] [Q&A] [answered]
I would like to know if the plugin rabbitmq-delayed-message-exchange supports rabbitmq-3.5.6?

**Comment by lukebakken** [comment, marked as answer]
Probably not. RabbitMQ 3.5.6 was released **over 7 years ago** and is **no longer supported**.

There's nothing stopping you from compiling this plugin yourself using whatever version of Erlang you're using for RabbitMQ 3.5.6
